packages/amplify-graphql-model-transformer/rds-lambda/interfaces/BaseRequest.ts

```diff
@@ -9,14 +9,14 @@ export interface RequestArgs extends BaseRequestArgs {
 
 export interface BaseRequest {
   table: string;
-  operation: 'GET' | 'LIST' | 'CREATE' | 'UPDATE' | 'DELETE';
+  operation: 'GET' | 'LIST' | 'CREATE' | 'UPDATE' | 'DELETE' | 'INDEX';
```

Comment:
`INDEX` doesn't feel like it's an operation. It's more like part of DDL.

Reply:
I'm going to cut a tagged release after this. I will clean up these method names in the next PR.